src/Public/Outputs.ps1

```diff
@@ -818,7 +818,7 @@ function Update-PodeWebSelect
             ID = $Id
             Options = $items
             DisplayOptions = @(Protect-PodeWebValues -Value $DisplayOptions -Default $items -EqualCount)
-            SelectedValue = [System.Net.WebUtility]::HtmlEncode($SelectedValue)
+            SelectedValue = $SelectedValue | ForEach-Object { [System.Net.WebUtility]::HtmlEncode($_) }
```

Comment:
The `Protect-PodeWebValues` helper would be better here:
```powershell
@(Protect-PodeWebValues -Value $SelectedValue -Encode)
```

Reply:
Thanks for the hint! PR was updated accordingly.